### ISSUE TITLE
Starting triton got stucked 

### ISSUE THREAD

**Issue by satchelwu** [triton-inference-server/server#2732]
=============================
== Triton Inference Server ==
=============================

NVIDIA Release 20.12 (build 18156940)

Copyright (c) 2018-2020, NVIDIA CORPORATION.  All rights reserved.

Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
NVIDIA modifications are covered by the license terms that apply to the underlying
project or file.

NOTE: The SHMEM allocation limit is set to the default of 64MB.  This may be
   insufficient for the inference server.  NVIDIA recommends the use of the following flags:
   nvidia-docker run --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 ...

I0415 03:37:08.720049 1 metrics.cc:221] Collecting metrics for GPU 0: Tesla V100-PCIE-32GB
I0415 03:37:09.643602 1 libtorch.cc:945] TRITONBACKEND_Initialize: pytorch
I0415 03:37:09.643636 1 libtorch.cc:955] Triton TRITONBACKEND API version: 1.0
I0415 03:37:09.643642 1 libtorch.cc:961] 'pytorch' TRITONBACKEND API version: 1.0
2021-04-15 03:37:10.064542: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
I0415 03:37:10.354688 1 tensorflow.cc:1877] TRITONBACKEND_Initialize: tensorflow
I0415 03:37:10.354727 1 tensorflow.cc:1887] Triton TRITONBACKEND API version: 1.0
I0415 03:37:10.354733 1 tensorflow.cc:1893] 'tensorflow' TRITONBACKEND API version: 1.0
I0415 03:37:10.354738 1 tensorflow.cc:1917] backend configuration:
{}
I0415 03:37:10.384291 1 onnxruntime.cc:1715] TRITONBACKEND_Initialize: onnxruntime
I0415 03:37:10.385118 1 onnxruntime.cc:1725] Triton TRITONBACKEND API version: 1.0
I0415 03:37:10.385137 1 onnxruntime.cc:1731] 'onnxruntime' TRITONBACKEND API version: 1.0
I0415 03:37:10.628039 1 pinned_memory_manager.cc:199] Pinned memory pool is created at '0x7fe4e8000000' with size 268435456
I0415 03:37:10.628433 1 cuda_memory_manager.cc:103] CUDA memory pool is created on device 0 with size 67108864
I0415 03:37:10.629171 1 backend_factory.h:44] Create TritonBackendFactory
I0415 03:37:10.630204 1 plan_backend_factory.cc:48] Create PlanBackendFactory
I0415 03:37:10.630218 1 plan_backend_factory.cc:55] Registering TensorRT Plugins
I0415 03:37:10.630260 1 logging.cc:52] Registered plugin creator - ::BatchTilePlugin_TRT version 1
I0415 03:37:10.630283 1 logging.cc:52] Registered plugin creator - ::BatchedNMS_TRT version 1
I0415 03:37:10.630293 1 logging.cc:52] Registered plugin creator - ::BatchedNMSDynamic_TRT version 1
I0415 03:37:10.630305 1 logging.cc:52] Registered plugin creator - ::CoordConvAC version 1
I0415 03:37:10.630312 1 logging.cc:52] Registered plugin creator - ::CropAndResize version 1
I0415 03:37:10.630318 1 logging.cc:52] Registered plugin creator - ::DetectionLayer_TRT version 1
I0415 03:37:10.630325 1 logging.cc:52] Registered plugin creator - ::FlattenConcat_TRT version 1
I0415 03:37:10.630341 1 logging.cc:52] Registered plugin creator - ::GenerateDetection_TRT version 1
I0415 03:37:10.630352 1 logging.cc:52] Registered plugin creator - ::GridAnchor_TRT version 1
I0415 03:37:10.630361 1 logging.cc:52] Registered plugin creator - ::GridAnchorRect_TRT version 1
I0415 03:37:10.630368 1 logging.cc:52] Registered plugin creator - ::InstanceNormalization_TRT version 1
I0415 03:37:10.630375 1 logging.cc:52] Registered plugin creator - ::LReLU_TRT version 1
I0415 03:37:10.630383 1 logging.cc:52] Registered plugin creator - ::MultilevelCropAndResize_TRT version 1
I0415 03:37:10.630390 1 logging.cc:52] Registered plugin creator - ::MultilevelProposeROI_TRT version 1
I0415 03:37:10.630397 1 logging.cc:52] Registered plugin creator - ::NMS_TRT version 1
I0415 03:37:10.630404 1 logging.cc:52] Registered plugin creator - ::Normalize_TRT version 1
I0415 03:37:10.630411 1 logging.cc:52] Registered plugin creator - ::PriorBox_TRT version 1
I0415 03:37:10.630418 1 logging.cc:52] Registered plugin creator - ::ProposalLayer_TRT version 1
I0415 03:37:10.630429 1 logging.cc:52] Registered plugin creator - ::Proposal version 1
I0415 03:37:10.630435 1 logging.cc:52] Registered plugin creator - ::PyramidROIAlign_TRT version 1
I0415 03:37:10.630445 1 logging.cc:52] Registered plugin creator - ::Region_TRT version 1
I0415 03:37:10.630451 1 logging.cc:52] Registered plugin creator - ::Reorg_TRT version 1
I0415 03:37:10.630461 1 logging.cc:52] Registered plugin creator - ::ResizeNearest_TRT version 1
I0415 03:37:10.630468 1 logging.cc:52] Registered plugin creator - ::RPROI_TRT version 1
I0415 03:37:10.630476 1 logging.cc:52] Registered plugin creator - ::SpecialSlice_TRT version 1
I0415 03:37:10.630482 1 logging.cc:52] Registered plugin creator - ::Split version 1
I0415 03:37:10.631508 1 custom_backend_factory.cc:46] Create CustomBackendFactory
I0415 03:37:10.631521 1 ensemble_backend_factory.cc:47] Create EnsembleBackendFactory
W0415 03:37:10.639432 1 model_repository_manager.cc:2020] ignore version directory '.vscode' which fails to convert to integral number
I0415 03:37:10.639468 1 model_repository_manager.cc:555] AsyncLoad() 'plate_license_recognition'
I0415 03:37:10.639496 1 model_repository_manager.cc:730] TriggerNextAction() 'plate_license_recognition' version 1: 1
I0415 03:37:10.639511 1 model_repository_manager.cc:768] Load() 'plate_license_recognition' version 1
I0415 03:37:10.639516 1 model_repository_manager.cc:787] loading: plate_license_recognition:1
I0415 03:37:10.739738 1 model_repository_manager.cc:846] CreateInferenceBackend() 'plate_license_recognition' version 1
I0415 03:37:10.749052 1 python.cc:566] 'python' TRITONBACKEND API version: 1.0
I0415 03:37:10.749076 1 python.cc:589] backend configuration:
{}
I0415 03:37:10.749297 1 python.cc:654] TRITONBACKEND_ModelInitialize: plate_license_recognition (version 1)
I0415 03:37:10.752172 1 model_config_utils.cc:1567] ModelConfig 64-bit fields:
I0415 03:37:10.752196 1 model_config_utils.cc:1569]     ModelConfig::dynamic_batching::default_queue_policy::default_timeout_microseconds
I0415 03:37:10.752202 1 model_config_utils.cc:1569]     ModelConfig::dynamic_batching::max_queue_delay_microseconds
I0415 03:37:10.752206 1 model_config_utils.cc:1569]     ModelConfig::dynamic_batching::priority_queue_policy::value::default_timeout_microseconds
I0415 03:37:10.752210 1 model_config_utils.cc:1569]     ModelConfig::ensemble_scheduling::step::model_version
I0415 03:37:10.752214 1 model_config_utils.cc:1569]     ModelConfig::input::dims
I0415 03:37:10.752218 1 model_config_utils.cc:1569]     ModelConfig::input::reshape::shape
I0415 03:37:10.752222 1 model_config_utils.cc:1569]     ModelConfig::model_warmup::inputs::value::dims
I0415 03:37:10.752227 1 model_config_utils.cc:1569]     ModelConfig::optimization::cuda::graph_spec::graph_lower_bound::input::value::dim
I0415 03:37:10.752231 1 model_config_utils.cc:1569]     ModelConfig::optimization::cuda::graph_spec::input::value::dim
I0415 03:37:10.752235 1 model_config_utils.cc:1569]     ModelConfig::output::dims
I0415 03:37:10.752239 1 model_config_utils.cc:1569]     ModelConfig::output::reshape::shape
I0415 03:37:10.752244 1 model_config_utils.cc:1569]     ModelConfig::sequence_batching::direct::max_queue_delay_microseconds
I0415 03:37:10.752248 1 model_config_utils.cc:1569]     ModelConfig::sequence_batching::max_sequence_idle_microseconds
I0415 03:37:10.752252 1 model_config_utils.cc:1569]     ModelConfig::sequence_batching::oldest::max_queue_delay_microseconds
I0415 03:37:10.752256 1 model_config_utils.cc:1569]     ModelConfig::version_policy::specific::versions
I0415 03:37:10.752432 1 python.cc:698] TRITONBACKEND_ModelInstanceInitialize: plate_license_recognition (GPU device 0)

I used the python backend, and "docker exec -it" info the container. A copy code of model.py was tested OK, but when triton started, It finally was stucked above.

**Comment by GuanLuo** [comment]
Can you run Triton with `--log-verbose=1` to see if it provides more information? And can you share a minimal reproduce step?

**Comment by satchelwu** [comment]
> Can you run Triton with `--log-verbose=1` to see if it provides more information? And can you share a minimal reproduce step?

I tried using “--log-verbose=1", but the stuck still exists.

**Comment by satchelwu** [comment]
In model.py,  "argparse.ArgumentParser" was invoked, I wonder if it is a point.

**Comment by satchelwu** [comment]
> In model.py, "argparse.ArgumentParser" was invoked, I wonder if it is a point.

That's it. Thanks.
